src/sql/src/session/vars.rs

```diff
@@ -1572,6 +1572,26 @@ pub const ENABLE_DEFAULT_CONNECTION_VALIDATION: ServerVar<bool> = ServerVar {
     internal: true,
 };
 
+pub static STATEMENT_LOGGING_MAX_DATA_CREDIT: Lazy<ServerVar<Option<usize>>> = Lazy::new(|| {
+    ServerVar {
+    name: UncasedStr::new("statement_logging_max_data_credit"),
+    value: None,
+    // The idea is that during periods of low logging, tokens can accumulate up to this value,
+    // and then be depleted during periods of high logging.
+    description: "The maximum number of bytes that can be logged for statement logging in short burts, or NULL if unlimited (Materialize).",
+    internal: false,
+}
+});
+
+pub static STATEMENT_LOGGING_TARGET_DATA_RATE: Lazy<ServerVar<Option<usize>>> = Lazy::new(|| {
+    ServerVar {
+    name: UncasedStr::new("statement_logging_target_data_rate"),
+    value: None,
```

Comment:
What's the reasoning behind defaulting to None instead of some high number?

Reply:
Basically just erring on the side of defaulting to keeping any new features off, but it's mostly arbitrary.